src/main/scala/info/bethard/timenorm/formal/Types.scala

```diff
@@ -489,15 +489,21 @@ case class AfterRI(interval: Interval,
 
 /**
   * Finds the interval between two input intervals. Formally:
+  * If inclusive:
+  * Between([t1,t2): startInterval,[t3,t4): endInterval): Interval = [t1,t4)
+  * Else:
   * Between([t1,t2): startInterval,[t3,t4): endInterval): Interval = [t2,t3)
   *
   * @param startInterval first interval
   * @param endInterval   second interval
+  * @param inclusive     include both intervals
   */
-case class Between(startInterval: Interval, endInterval: Interval) extends Interval {
+case class Between(startInterval: Interval, endInterval: Interval, inclusive: Boolean = false) extends Interval {
```

Comment:
You need separate `startIncluded` and `endIncluded` properties. Either one can be marked as included or excluded in the annotation.